### ISSUE TITLE
Clinkz hotkey

### ISSUE THREAD

**Issue by a100kpm** [ValveSoftware/Dota2-Gameplay#8461]
### Description

Windwalk and deathpact hotkey have been swapped in legacy

Burning barrage doesn't have an hotkey (not that it matter since no1 should buy/use that shard now that strafe is back)

**Comment by saglamhkn** [comment]
have same problem.

**Comment by izemar** [comment]
yeah, please fix it. we used to press W for invis and C for creep eating - it's been that way for years. temporary solution - unit specific hotkeys, brothers. and some D F for army and barrage
